git_machete/client.py

```diff
@@ -1852,7 +1852,8 @@ def checkout_github_prs(self,
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
         if not current_user and my_opened_prs:
-            msg = f"Could not determine current user name, please check your {GITHUB_TOKEN_ENV_VAR} environment variable."
+            msg = ("Could not determine current user name, please check that the GitHub API token provided by one of the: "
```

Comment:
Could you show a screenshot of how this message looks in the terminal now?

Reply:
<img width="1340" alt="image" src="https://user-images.githubusercontent.com/86779115/161507869-e0d94475-f861-4451-9e7a-a029f1f3ab37.png">